src/pystitcher/stitcher.py

```diff
@@ -24,6 +24,10 @@ def __init__(self, inputBuffer):
         DEFAULT_FIT = '/FitV'
         # Do not rotate by default
         DEFAULT_ROTATE = 0
+        # Start at page 1 by default
+        DEFAULT_START = 1
+        # End at 0 pages before the final page by default
+        DEFAULT_END = 0
```

Comment:
Not sure about the double definition of `end` (user input is interpreted as ending page, but default input is negative page count).

A special value of `None` to avoid that confusion?